test/integration/targets/setup_podman/tasks/main.yml

```diff
@@ -13,10 +10,6 @@
 
     - name: Get podman version
       command: podman --version
-      register: podman_version
```

Comment:
Worth to leave it, it's always good to know on which version it runs.

Reply:
Since we run with `-v` by default, this information is already displayed, so there is no need to do the register/debug dance.